packages/flutter/lib/src/material/drawer.dart

```diff
@@ -118,13 +133,15 @@ class DrawerController extends StatefulWidget {
   const DrawerController({
     GlobalKey key,
     @required this.child,
-  }) : assert(child != null),
+    @required this.type,
+  }) : assert(child != null || type != null),
        super(key: key);
 
   /// The widget below this widget in the tree.
   ///
   /// Typically a [Drawer].
   final Widget child;
+  final DrawerAlignment type;
```

Comment:
missing dartdoc

Reply:
Also, should probably be named "alignment" now that the type is called DrawerAlignment rather than DrawerType

Reply:
Fixed